models/index.js

```diff
@@ -39,4 +39,4 @@ User_Subscription.belongsTo(User, {
     foreignKey: 'user_id'
 })
 
-module.exports = { Service, Show, User, User_Subscription };
+module.exports = { Service, TvShow: TvShow, User, User_Subscription };
```

Comment:
I think this can just be TvShow not TvShow: TvShow ?

Reply:
TvShow is just short hand, is this going to block the PR?